submarine-commons/commons-runtime/src/main/java/org/apache/submarine/commons/runtime/fs/DefaultRemoteDirectoryManager.java

```diff
@@ -134,13 +134,13 @@ public boolean copyRemoteToLocal(String remoteUri, String localUri)
   }
 
   @Override
-  public boolean existsRemoteFile(Path url) throws IOException {
-    return getFileSystemByUri(url.toUri().toString()).exists(url);
+  public boolean existsRemoteFile(String remoteUri) throws IOException {
+    return getFileSystemByUri(remoteUri).exists(new Path(remoteUri));
   }
 
   @Override
-  public FileStatus getRemoteFileStatus(Path url) throws IOException {
-    return getFileSystemByUri(url.toUri().toString()).getFileStatus(url);
+  public FileStatus getRemoteFileStatus(String remoteUri) throws IOException {
```

Comment:
We'd better use change getRemoteFileStatus(String remoteUri) to getRemoteFileStatus(String uri). So that the parameter name uri is the same as the one of the interface RemoteDirectoryManager

Reply:
Hi @yuanzac 

Thanks for the review. I updated the part you mentioned.